src/accounts.js

```diff
@@ -4,6 +4,12 @@ const ACCOUNT_NAME_MAX_LENGTH = 12;
 const BASE = 31; // Base 31 allows us to leave out '.', as it's used for account scope
 
 /* Private */
+// gets the input values(blocksToCheck, checkInterval, getBlockAttempts) required for the awaitTranaction method
+function getAwaitTransactionOptions(options){
+const awaitTransactionOptions = {};
+({blocksToCheck: awaitTransactionOptions.blocksToCheck, checkInterval: awaitTransactionOptions.checkInterval, getBlockAttempts: awaitTransactionOptions.getBlockAttempts} = options);
```

Comment:
Yes, its needed.  If omitted, the destructuring object will be taken to be a block statement, which will lead to an error because a block cannot appear at the left-hand-side of an assignment expression.